aten/src/ATen/native/native_functions.yaml

```diff
@@ -2989,22 +2989,29 @@
 - func: quantize_linear(Tensor self, float scale, int zero_point) -> Tensor
   matches_jit_signature: True
   variants: function, method
-  requires_tensor: True
+  dispatch:
```

Comment:
I see that quantize_linear is replaced with quantize_linear_cpu in QTensor.cpp. Should this declaration also change?

Reply:
declaration is exposed in python frontend, so we'll use quantize_linear